test/k4FWCoreTest/src/components/k4FWCoreTest_cellID_reader.cpp

```diff
@@ -41,10 +43,10 @@ StatusCode k4FWCoreTest_cellID_reader::initialize() {
 StatusCode k4FWCoreTest_cellID_reader::execute(const EventContext&) const {
   const auto simtrackerhits_coll = m_simTrackerHitReaderHandle.get();
 
-  auto       collID    = simtrackerhits_coll->getID();
-  const auto cellIDstr = m_cellIDHandle.get();
+  const auto cellIDstr = m_cellIDHandle.get("");
```

Comment:
```suggestion
  const auto cellIDstr = m_cellIDHandle.get();
```

This allows us to differentiate between error in retrieval, and retrieval of wrong value.